tests/pipeline/test_blaze.py

```diff
@@ -1480,22 +1575,42 @@ def test_novel_deltas(self, asset_info):
             name='delta',
             dshape=self.dshape,
         )
-        expected_views = keymap(pd.Timestamp, {
+        expected_views_all_deltas = keymap(pd.Timestamp, {
             '2014-01-03': np.array([[10.0, 11.0, 12.0],
                                     [10.0, 11.0, 12.0],
                                     [10.0, 11.0, 12.0]]),
             '2014-01-06': np.array([[10.0, 11.0, 12.0],
                                     [10.0, 11.0, 12.0],
                                     [11.0, 12.0, 13.0]]),
         })
+        expected_views_novel_deltas = keymap(pd.Timestamp, {
+            '2014-01-03': np.array([[0.0, 1.0, 2.0],
+                                    [0.0, 1.0, 2.0],
+                                    [0.0, 1.0, 2.0]]),
+            '2014-01-06': np.array([[0.0, 1.0, 2.0],
+                                    [0.0, 1.0, 2.0],
+                                    [11.0, 12.0, 13.0]]),
```

Comment:
By the design of this test, that's the only novel delta here. It's the only case where the base data point came in on a weekend (Saturday, 2014-01-04) and the delta came in the day after, on a Sunday, so that it replaced the most recently known value before we would have had a chance to see it. The other deltas are for data points that we would have seen before seeing the deltas, so they're not novel deltas.